packages/squiggle-lang/__tests__/Distributions/Symbolic_test.res

```diff
@@ -39,24 +39,24 @@ describe("(Symbolic) mean", () => {
   })
 
   testAll("of exponential distributions", list{1e-7, 2.0, 10.0, 100.0}, rate => {
-    let meanValue = run(FromDist(ToFloat(#Mean), GenericDist_Types.Symbolic(#Exponential({rate: rate}))))
+    let meanValue = run(FromDist(ToFloat(#Mean), DistributionTypes.Symbolic(#Exponential({rate: rate}))))
     meanValue -> unpackFloat -> expect -> toBeCloseTo(1.0 /. rate)  // https://en.wikipedia.org/wiki/Exponential_distribution#Mean,_variance,_moments,_and_median
   })
 
   test("of a cauchy distribution", () => {
-    let meanValue = run(FromDist(ToFloat(#Mean), GenericDist_Types.Symbolic(#Cauchy({local: 1.0, scale: 1.0}))))
+    let meanValue = run(FromDist(ToFloat(#Mean), DistributionTypes.Symbolic(#Cauchy({local: 1.0, scale: 1.0}))))
     meanValue
     -> unpackFloat
     -> expect
-    -> toBeCloseTo(2.01868297874546)
+    -> toBeCloseTo(1.0098094001641797)  // 2.01868297874546)
```

Comment:
Notice this number changed when I changed the `sampleCount` value in `TestHelpers.env`. 

In general the `cauchy` mean situation is not great, I'm not happy with it.